.github/pull_request_template.md

```diff
@@ -4,17 +4,22 @@ Resolves # (*If this PR is unrelated to a ticket, please erase this line*)
 
 Please include a summary of the change
 
-
 ### Type of change
 
-Please choose one of the following: Bug fix, new feature, new documentation, or other (please explain).
+Please check one of the following:
+* [ ] Bug fix
+* [ ] New feature
+* [ ] Documentation update
+* [ ] Other (please explain with a note below)
 
 ### How Has This Been Tested?
 
 Please describe the tests you've added or any tests that already cover this change. Include relevant information, such as which operating system you tested on.
 
 ### Checklist:
 
+For items in this checklist that do not apply, simply insert two tilde chars, `~~`, just ahead of the left bracket char, `[` at the beginning of a line. Each line ends with two tilde chars to make doing such ~~strikeouts~~ easy.
```

Comment:
You can make this a comment so that it only appears in the editable text box.

Reply:
`<!--This is how you make a comment, just like in HTML-->`

Reply:
Hmmm....I kinda like that I guess but why wouldn't we do that for *all* of the instructional text in the PR template. So, also, for example, lines like this one...

<!-- Please include a summary of the change -->

Reply:
Doing it in an XML comment does also elim. use of any markdown in the comment. Which is ok too but just wanted to mention.

Reply:
Good point. I'm in favor of commenting all the instructional text.